rhemyn/compile.js

```diff
@@ -43,14 +43,15 @@ const generate = (node, negated = false, get = true, stringSize = 2, func = 'tes
             [ Opcodes.return ],
           [ Opcodes.end ],
 
-          // increment counter by 1, check if eq length, if not loop
+          // counter++, if length > counter, loop
+          [ Opcodes.local_get, Length ],
+
           [ Opcodes.local_get, Counter ],
           ...number(1, Valtype.i32),
           [ Opcodes.i32_add ],
           [ Opcodes.local_tee, Counter ],
 
-          [ Opcodes.local_get, Length ],
-          [ Opcodes.i32_ne ],
+		  [ Opcodes.i32_gt_s ],
```

Comment:
Sorry about that, I had an .editorconfig that was managing that for me before, and had removed it since then. It's a bit unrelated, but maybe adding one could be helpful for more than just me? What are your thoughts?
Regardless, I'll go ahead and fix this.

Reply:
> It's a bit unrelated, but maybe adding one could be helpful for more than just me? What are your thoughts?

Good idea, will do later :+1: